<?xml version="1.0"?>

<launch>

	<node pkg="robot_pose_ekf" type="robot_pose_ekf" name="robot_pose_ekf">
	  <param name="output_frame" value="odom_combined"/>
	  <param name="base_footprint_frame" value="angelbot_base"/>
	  <param name="freq" value="10.0"/>
	  <param name="sensor_timeout" value="1.0"/>  
	  <param name="odom_used" value="true"/>
	  <param name="imu_used" value="true"/>
	  <param name="vo_used" value="false"/>
	  <param name="publish_tf" value="true"/>
	<!--
	 <remap from="odom" to="pr2_base_odometry/odom" />
	 -->
	  <remap from="odom" to="angelbot/odom" />
	  <remap from="robot_pose_ekf/odom_combined" to="angelbot/odom_combined"/>
	
	</node>
	<!-- node pkg="robot_pose_ekf" type="gyro_node.py" name="gyro_node"/-->
	<node pkg="robot_pose_ekf" type="mpu6050.py" name="mpu6050" />		
	<node pkg="robot_pose_ekf" type="odom_ekf.py" name="odom_ekf" />

</launch>

